seed/views/columns.py

```diff
@@ -149,6 +149,20 @@ def retrieve(self, request, pk=None):
             'column': ColumnSerializer(c).data
         })
 
+    @ajax_request_class
+    @has_perm_class('can_modify_data')
+    def update(self, request, pk=None):
+        organization_id = request.query_params.get('organization_id', None)
+
+        request.data['shared_field_type'] = request.data['sharedFieldType']
+        del request.data['sharedFieldType']
+
+        # Ensure ComStock uniqueness across properties and taxlots together
+        if request.data['comstock_mapping'] is not None:
+            Column.objects.filter(organization_id=organization_id, comstock_mapping=request.data['comstock_mapping']) \
+                .update(comstock_mapping=None)
```

Comment:
Ah I see - I was looking for this dedup on the frontend.  My only hesitation with this approach is that we don't indicate what's happening on the frontend. 

Brainstorming some:
- What if we updated the frontend to expand the scope of `$scope.columns` to include both inventories? We'd be able to track and dedup the comstock_mapping value across the whole org there. We'd probably have to update the html partial to only display columns of a the given inventory type. Also, we'd have to update the confirm save modal to show inventory type since we could be on the property page but change a tax lot column.
- Would it be enough to modify the saved changes notifications to reflect that another column was changed? This approach would probably require us to edit the response coming out of this method. Initial feeling is that this wouldn't be a great approach.

Reply:
The confirm modal now shows `(removed)` for comstock mappings that already existed on the other table 👍